docs/choreolib/auto-routines.md

```diff
@@ -89,26 +89,26 @@ public AutoRoutine fivePieceAutoTriggerSeg(AutoFactory factory) {
   final AutoTrajectory s1ToC2 = factory.trajectory("s1ToC2", routine);
   final AutoTrajectory c2ToC3 = factory.trajectory("c2ToC3", routine);
 
+  final Optional<Pose2d> initialPose = ampToC1.getInitialPose();
+  if (initialPose.isEmpty()) {
+      new Alert("Error: 5 piece auto has no starting pose", AlertType.kError)
+          .set(true);
+      return factory.voidRoutine();
+  }
+
   // entry point for the auto
   // resets the odometry to the starting position,
   // then shoots the starting note,
   // then runs the trajectory to the first close note while extending the intake
   routine.running()
       .onTrue(
-          resetOdometry(
-                  ampToC1
-                      .getInitialPose()
-                      .orElseGet(
-                          () -> {
-                            routine.kill();
-                            return new Pose2d();
-                          }))
+          resetOdometry(initialPose.get())
               .andThen(
                   autoAimAndShoot(),
                   race(
                       intake(),
                       ampToC1.cmd(),
-                      aimFor(ampToC1.getFinalPose().orElseGet(Pose2d::new))))
+                      aimFor(ampToC1.getFinalPose().orElse(new Pose2d()))))
```

Comment:
Idk how this even got here but it shouldn't be here

Reply:
No yeah i replaced it on purpose; i think i should revert it tho